src/lib/plugin.js

```diff
@@ -379,8 +379,8 @@ class FiveBellsLedger extends EventEmitter2 {
     if (!res || res.statusCode !== 200) { throwErr() }
 
     // note that the fivebells ledger API uses 'scale' instead of 'currency_scale'.
-    // the 'ilp_prefix' field is not mandatory
-    if (!res.body.currency_code || !res.body.scale) { throwErr() }
+    // fields like 'ilp_prefix' and 'currency_code' are not mandatory
+    if (!res.body.connectors || !res.body.scale) { throwErr() }
```

Comment:
@dappelt this was making the 'index' integration test fail (since the currency_code of `test1.ledger1` is null there), is that not something you ran into?